Frontend/CO.CDP.OrganisationApp/Constants/ApiExceptionMapper.cs

```diff
@@ -17,9 +17,11 @@ public static void MapApiExceptions(
             return;
         }
 
-        var errorMessage = GetErrorMessageByCode(code);
+        // Capture the exception message
+        var exceptionMessage = aex.Result.Detail;
+
+        var errorMessage = GetErrorMessageByCode(code, exceptionMessage);
```

Comment:
I'm surprised this is the first time we needed to do this

Reply:
yes, it is first time and I have to twick it.